app/views/provider/admin/backend_apis/forms/_delete.html.slim

```diff
@@ -6,16 +6,27 @@
   p
     ' Deleting this backend API will
     strong> irreversibly
-    ' destroy all methods & metrics and mapping rules of this backend API and remove it from all products currently using the backend API.
+    ' destroy all methods & metrics and mapping rules of this backend API.
     ' It will also delete in the the application plans all limits and pricing rules set on methods & metrics of this backend API.
 
   p
     strong> Important!
     ' Proxy configurations of products using this backend API (staging and production) will not update automatically after deleting the backend API.
     ' You need to perform this action on each product using the backend, manually via UI or 3scale API.
 
-  p
-    - msg = t('api.backend_apis.edit.delete_confirmation', name: h(backend_api.name))
-    - delete_options = { data: { confirm: msg }, method: :delete, label: "I understand the consequences, proceed to delete '#{h(backend_api.name)}' backend" }
-    = delete_link_for(provider_admin_backend_api_path(backend_api), delete_options)
+  - if backend_api.backend_api_configs.any?
+    p The following products are using this backend API:
+    ul
+      - for config in backend_api.backend_api_configs
```

Comment:
This will trigger a N+1 requests so suggesting 
```suggestion
      - backend_api.services.each do |service|
```

Reply:
Would `- for service in backend_api.services` be equivalent?